### ISSUE TITLE
Question: How to NOT generate a requirements.txt. Force to use my requirements.txt

### ISSUE THREAD

**Issue by grayaii** [UnitedIncome/serverless-python-requirements#405]
We have a requirements.txt that has the specific ordering of pip installs.
It works great if you do a: pip install -r requirements.txt.

It looks like this plugin is GENERATING a new requirements.txt file:

Serverless: Generated requirements from /code/requirements.txt in /code/.serverless/requirements.txt...

And that generated requirements.txt file is out of order and thus fails.

Can we tell this plugin to NOT generate a requirements.txt?
In other words, can we tell this plugin to use my specific requirements.txt?

Here are the options we are using:

```
custom:
  pythonRequirements:
    dockerizePip: false
    useDownloadCache: true
    useStaticCache: true
    slim: true
    usePipenv: false
```

**Comment by BerserkerGaruk** [comment]
Does the fileName arguement work for you?:
```
pythonRequirements:
    fileName: requirements.txt
```

**Comment by grayaii** [comment]
@BerserkerGaruk Yes! That option disables the auto-creation of the requirements.txt file.
Thanks!
